project/Easy-Travel-BD/EasyTravel.Web/wwwroot/assets/vendor/aos/aos.esm.js

```
@@ -0,0 +1,610 @@
+import throttle from 'lodash.throttle';
+import debounce from 'lodash.debounce';
+
+var callback = function callback() {};
+
+function containsAOSNode(nodes) {
+  var i = void 0,
+      currentNode = void 0,
+      result = void 0;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)